torchvision/transforms/functional.py

```diff
@@ -223,10 +223,10 @@ def to_pil_image(pic, mode=None):
             pic = np.expand_dims(pic, 2)
 
     npimg = pic
-    if isinstance(pic, torch.FloatTensor) and mode != 'F':
-        pic = pic.mul(255).byte()
     if isinstance(pic, torch.Tensor):
-        npimg = np.transpose(pic.numpy(), (1, 2, 0))
+        if pic.is_floating_point() and mode != 'F':
```

Comment:
note: this seems to actually fix a bug where if the image was `double` it wouldn't be multiplied by 255 and cast to byte

Reply:
Could you please develop this note. Do we need to do something here ?
I'm writing a follow-up issue to address the improvements...

Reply:
Just making sure we properly track it somehow so that we can mention this in the release notes